tests/unit/ui/renderers/tools/test_renderer_write_web_update.py

```
@@ -0,0 +1,481 @@
+        )
+        assert "lines" in status.plain
+
+    def test_border_color_always_success(self) -> None:
+        color = self.renderer.get_border_color(self.data)
+        assert color == self.renderer.config.success_color
+
+    # -- full render --
+
+    def test_render_valid_result(self) -> None:
+        args: dict[str, Any] = {
+            "filepath": "/tmp/new.py",
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/new.py"

(S108)